wiki.go

```diff
@@ -98,6 +98,8 @@ func rootHandler(w http.ResponseWriter, r *http.Request) {
 func main() {
 	http.HandleFunc("/", rootHandler)
 	http.HandleFunc("/view/", makeHandler(viewHandler))
+	http.HandleFunc("/edit/", makeHandler(editHandler))
+	http.HandleFunc("/save/", makeHandler(saveHandler))
```

Comment:
You may not want edit and save options here. This is a long lived app on the cluster and anyone with an internet connection can edit and save.